File: Data Structures & Algorithms/is-anagram/submission-1.java
class Solution {
    public boolean isAnagram(String s, String t) {
    int m=s.length();
        int n=t.length();
        if(m!=n){
            return false;
        }
        int [] cnt=new int [26];
        for(int i=0;i<m;i++){
            cnt[s.charAt(i)-'a']++;
        }
        for(int j=0;j<n;j++){
            cnt[t.charAt(j)-'a']--;
        }
        for(int i=0;i<cnt.length;i++){
            if(cnt[i]!=0){
                return false;
            }
            
        }
        return true;

    }
}

File: Data Structures & Algorithms/majority-element/submission-1.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        cnt=0
        elem=0
        for i in nums:
            if cnt==0:
                cnt=1
                elem=i
            elif i==elem:
                cnt+=1
            else:
                cnt-=1
        return elem

        